bin/laminas

```diff
@@ -23,8 +27,33 @@ if (file_exists($a = __DIR__ . '/../../../autoload.php')) {
     exit(1);
 }
 
+$projectRoot = dirname($a) . '/../';
+
 // Set the main application directory as the current working directory
-chdir(dirname($a) . '/../');
+chdir($projectRoot);
+
+$app = (new ApplicationFactory())();
+$definition = $app->getDefinition();
+$output = new ConsoleOutput();
+$containerNotFoundMessage = '';
+try {
+    $input = new ArgvInput();
+    $input->bind($definition);
+    $container = (new ContainerResolver($projectRoot))->resolve($input);
+    $app = (new ApplicationConfigurator())($app, $container);
+} catch (\Symfony\Component\Console\Exception\RuntimeException $exception) {
+    // Suppress as it will be part of the application output
+} catch (RuntimeException | InvalidArgumentException $exception) {
+    $containerNotFoundMessage = sprintf('<error>%s</error>', $exception->getMessage());
+} finally {
+    $input = new ArgvInput();
+}
+
+$exitCode = $app->run($input, $output);
```

Comment:
Why are we running the app if the container is not found?

Reply:
The main idea behind this is, that you are able to see `--help` where all available options are listed (even tho, the new `--container` option).

Reply:
Thats why I provide the error message that the container could not be found afterwards.

Reply:
Makes sense. Again, I'd drop a note in here along the lines of "Running the app after suppressing container resolution exceptions allows symfony/console to report usage information".